Tests/Utility Tests/DateFormatTests.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  DateFormatTests.swift
+//  
+//
+//  Created by Brandon Walton on 1/20/22.
+//
+
+import XCTest
+@testable import Guppy
+
+class DateFormatTests: XCTestCase {
+
+    let testDate = Calendar.current.date(from: DateComponents(year: 1995, month: 1, day: 1, hour: 1, minute: 25, second: 0))!
+    
+    func testSharedFormatter() throws {
+        DateFormat.shared.setStyle(time: .medium, date: .medium)
```

Comment:
This will fail if run in a different locale. Can we change the test or explicitly set the locale?

Reply:
Good catch!